modules/javafx.graphics/src/test/java/test/javafx/scene/layout/BackgroundTest.java

```diff
@@ -714,6 +717,13 @@ public void imagesIsUnmodifiable() {
         assertFalse(b.isFillPercentageBased());
     }
 
+    @Test
+    public void testSingleFill() {
+        var background1 = Background.fill(Color.BEIGE);
```

Comment:
I think we should have a test for null argument too.
Similar for the test BorderTest.testSingleStroke()

Reply:
Yes, I wanted to add these but forgot.